wrappers/nodejs/test/test-devicelist.js

```diff
@@ -45,6 +45,32 @@ describe('DeviceList test', function() {
     assert.equal(typeof devl.size, 'number');
   });
 
+  it('Testing member back', () => {
+    assert.doesNotThrow(() => {
+      if (devl.size > 0) {
+        let res = devl.back;
+        assert(devl.back instanceof rs2.Device);
+        assert.equal(typeof res, 'object');
```

Comment:
Besides the typeof and instanceof, we should also check if this device is exactly the first(or last) device.
e.g. You can check both devices' serial number.